packages/eth-sdk/src/config/readConfig.ts

```diff
@@ -31,7 +31,7 @@ async function registerTsNode() {
   try {
     const { register } = await import('ts-node')
 
-    register({ compilerOptions: { module: 'CommonJS' } })
+    register({ compilerOptions: { module: 'CommonJS' }, transpileOnly: true })
```

Comment:
I think we should warn users about problems with the config. Is there any reason to have this turned on? Did you figure out the problem with the "double" transpilation?